data/hlint.yaml

```diff
@@ -367,6 +368,8 @@
     - hint: {lhs: pure x <*> y, rhs: x <$> y}
     - warn: {lhs: x <* pure y, rhs: x}
     - warn: {lhs: pure x *> y, rhs: "y"}
+    - hint: {lhs: zip (map f x) (map g x), rhs: map (liftA2 (,) f g) x}
```

Comment:
Not convinced on this one.

Reply:
Yeah, `&&&` is the one thing arrows offer without a nice clean replacement. Instead of `liftA2 (,) f g`, we could always do `(,) <$> f <*> g`, but I feel like you'd dislike that even more. I may just remove these ones.

Reply:
Yeah, I think saying nothing is probably best.